js/suitcase_packing.js

```diff
@@ -0,0 +1,10 @@
+//https://www.codewars.com/kata/suitcase-packing/train/javascript
+
+function fit_in(a, b, m, n) {
+  if((Math.pow(a,2) + Math.pow(b,2)) <= m * n){
```

Comment:
Can this be simplified into a single `return`?

Consider how to simplify:

```ts
if (true) {
  return true;
} else {
  return false;
}
```

Reply:
Done :)